src/main/java/org/folio/des/controller/FolioTenantController.java

```diff
@@ -60,4 +108,314 @@ public ResponseEntity<Void> postTenant(TenantAttributes tenantAttributes) {
     return tenantInit;
   }
 
+  private boolean isLegacyJob(LegacyJob job) {
```

Comment:
Can we extract some of the legacy job logic into a separate method somewhere?

Reply:
@ncovercash Yeah, it did feel out of place but I wasn't sure where to put it. What do you think about moving it to a `LegacyBursarUtil` class in the `util` package?

Reply:
`util` package sounds great, I'd recommend maybe something like `LegacyBursarMigrationUtil` for clarity tho

Reply:
Just took care of that!